cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -162,6 +166,28 @@ public BodyConsumer deploy(HttpRequest request, HttpResponder responder,
   }
 
   /**
+   * Creates an application with the specified name and app-id from an artifact.
+   */
+  @POST
+  @Path("/apps/new/{app-id}/versions/{version-id}/create")
```

Comment:
yes, this is just a temporary fix in order to run the test and validate the changes for it